lib/coinbase.rb

```diff
@@ -14,18 +14,27 @@
 
 # The Coinbase SDK.
 module Coinbase
-  @base_sepolia_rpc_url = 'https://sepolia.base.org'
+  def self.configuration
+    @configuration ||= Configuration.new
+  end
 
-  # Returns the Base Sepolia RPC URL.
-  # @return [String] the Base Sepolia RPC URL
-  def self.base_sepolia_rpc_url
-    @base_sepolia_rpc_url
+  def self.configure
+    yield(configuration)
   end
 
-  # Sets the Base Sepolia RPC URL.
-  # @param value [String] the Base Sepolia RPC URL
-  def self.base_sepolia_rpc_url=(value)
-    @base_sepolia_rpc_url = value
+  # Configuration object for the Coinbase SDK
```

Comment:
Could you please scope in api_key_name and api_key_private_key below?

Reply:
And all of the other api clients as well (users_api, wallets_api, addresses_api)

Reply:
Yes!!!